lua/fzf-lua-frecency/algo.lua

```diff
@@ -63,25 +62,24 @@ M.update_file_score = function(filename, opts)
 
   local fs = require "fzf-lua-frecency.fs"
   local h = require "fzf-lua-frecency.helpers"
-  local cwd = h.default(opts.cwd, vim.fn.getcwd())
+  local db_index = 1 -- We only use index 1 everywhere
```

Comment:
Do we need `db_index` at all? I think we can remove it and use `dated_files[filename]` directly